src/main/java/uk/gov/companieshouse/web/pps/controller/pps/ConfirmationController.java

```diff
@@ -113,17 +126,15 @@ public String getConfirmation(@PathVariable String companyNumber,
 
     private String setUpPaymentDateDisplay(PayableLateFilingPenalty payableLateFilingPenalty) {
```

Comment:
Could this method be pulled out into the 'PenaltyUtils'? For me there is too much logic in the controller. I am hoping that we create a service layer for all this post the first release.